final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,143 @@
+import feedparser
+import sys
+import datetime
+import argparse
+import json
+from bs4 import BeautifulSoup
+
+DEFAULT_LIMIT_COUNT = 3
+VERSION = '1.0'
+
+
+class RSSReader:
+    is_verbose = False  # logging is disabled by default
+    is_json = False
+    limit = DEFAULT_LIMIT_COUNT
+
+    def __init__(self, url):
+        self.url = url
+
+    def url_parsing(self):
+        parsed_url = feedparser.parse(self.url)
+        if parsed_url.status != 200:  # URL-access check
+            print("Can't load RSS feed.")
+            raise Exception('RSS-reader failed. Status: {}.'.format(parsed_url.status))
```

Comment:
Minor thing, but it is better to make your own custom exception here which inherits `Exception` class